server/testserver.go

```diff
@@ -83,6 +85,11 @@ func (ts *TestServer) Start() error {
 	if ts.Addr == "" {
 		ts.Addr = "127.0.0.1:0"
 	}
+	if ts.CertDir == "" {
+		// Always load server certs. Not sure if this path is a good assumption though,
+		// something relative might be best.
+		ts.CertDir = path.Join(os.ExpandEnv("$GOPATH"), "src/github.com/cockroachdb/cockroach/resource/test_certs")
```

Comment:
Use os.Getenv("GOPATH") instead of os.ExpandEnv("$GOPATH").

We can't use a relative path because server.TestServer is used for tests starting in different directories (we could have TestServer.Start() take an argument for the directory the test is being run from, but that's ugly too). This will work for now (we have some other assumptions about $GOPATH elsewhere in the codebase) but I think ideally we'd have different cert Loaders similar to the bootstrap Resolvers, and one of them could load certs baked into the binary with go-bindata.